imls-wifi-sensor/README.md

```diff
@@ -1,8 +1,22 @@
 # Wifi Sensor
 
-## Running session-counter
 Assumes you are a developer with git and Golang installed
 
+## Changing the Time to Send Data
+Modify session-counter.ini in imls-wifi-sensor/cmd/linux-session-counter and/or imls-wifi-sensor/cmd/windows-session-counter
+
+- If in prod:
+  - [cron]
+    reset=0 0 ** *
```

Comment:
I think we need to wrap the cron statements in blockquotes, otherwise the `*` gets interpreted as markdown

Reply:
or do `reset=0 0 * * *` -- either way, we're accidentally interpreting this as markdown `**` is bold, as **in bold**

Reply:
Good catch, thanks!